### PR TITLE
Add ppc64le build

### PR DESCRIPTION
**Pull request by tschoonj**

### CODE REVIEW

**Comment by ryanvolz** [comment]
I think you can relax the `gdk_pixbuf` pin so that we don't have to keep updating it. This change will probably come to the global pinnings: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/774.

**Comment by tschoonj** [comment]
Yeah, the gdk-pixbuf pinning is wrong. 

Actually what I think will fix a lot here is apply the Glib 2.64.6 [migration](https://conda-forge.org/status/#glib2646) to gdk-pixbuf, and then to gtk3. For some reason this never happened, even though all their parents are fine already. Is there an elegant way to force this?

**Comment by ryanvolz** [comment]
Looks like the migration is held up by Qt, which is really unfortunate for the *other* main graphical toolkit. Other than breaking that dependency chain somehow I don't know if there's a way around this. I'll give it some thought.

**Comment by ryanvolz** [comment]
The bot's migration graph shows a lot of things that depend on `opencv` (or perhaps one of its build outputs instead, e.g. `libopencv`) which depends on `qt`. One of those things is `gdk-pixbuf`, but I don't see from the recipe where it gets that idea. If the bot is wrong, then manually migrating `gdk-pixbuf` to the new `glib` will let us progress.

**Comment by ryanvolz** [comment]
Oh duh, you already did the migration manually on `gdk-pixbuf`! You can do the same here, and hopefully that will clear it up. I still don't understand why the bot wants to migrate `opencv` first...

**Comment by tschoonj** [comment]
Oh yes I did 😄 

Ok, I will do the same thing here then.

**Comment by ryanvolz** [comment]
I pinged core about updating the gdk-pixbuf global pin in conda-forge-pinnings, but for this to work for now you'll have to add [this migration](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/deda6f9e8823075e9d6a733cd99627fa22843b81/recipe/migrations/gdk-pixbuf_max_pin_2.yaml) manually as well.

**Comment by tschoonj** [comment]
Not sure what the effect of that migration will be. We really need to use gdk-pixbuf 2.42.0 from now on as [that build](https://github.com/conda-forge/gdk-pixbuf-feedstock/commit/5a4f94254c26b8c9616d81469530becf6aebc880#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9a) removes gobject-introspection from its runtime requirements.

The fact that the Linux/macOS builds succeeded with my latest commit here involved a bit of luck as this could easily have gone wrong as it did with the earlier builds.

**Comment by ryanvolz** [comment]
It would override the global pinning on gdk-pixbuf 2.36.12, which would then allow 2.42.0 to be used for the build since it is the newest. The only problems with gdk-pixbuf are at build time because of that old global pin, which needs to be updated to `2` to match the fact that all versions with that major version of `2` are ABI-compatible. Better, perhaps, than the migration, is to just pin `gdk-pixbuf` to `2` in `conda_build_config.yaml`.

**Comment by tschoonj** [comment]
Ok thanks, will give that a try

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,3 +1,2 @@
-# update gdk_pixbuf pin for windows (version from conda-forge-pinning doesn't exist)
-gdk_pixbuf:  # [win]
-  - 2.38.2  # [win]
+gdk_pixbuf:
+  - 2.38.2
```
Comment (ryanvolz) [inline recipe/conda_build_config.yaml:2]: Just `2` here will do it.

**Comment by tschoonj** [comment]
yep, that was some bad copy-pasting...

**Comment by ryanvolz** [comment]
The global pinning was just updated thanks to my prodding, so the local `conda_build_config.yaml` won't be needed for much longer anyhow. It's still fine to have it for this PR (and hopefully it works), but we'll just have to remember to remove it before `gdk-pixbuf` reaches version 3!

**Comment by ryanvolz** [comment]
The dependency tree is going to be much cleaner without `gobject-introspection` in `host` by the way, extra thanks for pushing that along! I'm trying to think if there are other packages that will need to do the same.

**Comment by ryanvolz** [comment]
> I'm trying to think if there are other packages that will need to do the same.

at-spi-atk
at-spi-core
librsvg
pango
pygobject

**Review by ryanvolz — APPROVED** [review]

**Comment by tschoonj** [comment]
Thanks! Next on my list is pygobject...

**Comment by ryanvolz** [comment]
> Thanks! Next on my list is pygobject...

Yes, that one in particular caught my eye as well. I'm sure you saw https://github.com/conda-forge/gobject-introspection-feedstock/issues/32 which I just created. Until something like that is done, it seems like you'll be able to move `gobject-introspection` from host to build for `pygobject` but will need to keep it in `run`.

**Pull request merged**